packages/web-components/src/stories/popup.stories.mdx

```diff
@@ -32,25 +32,24 @@ A Pop-Up Menu provides users with a quick way to access common actions for a hig
 
 export const Default = (args) => {
     return html`
-        <div style="display: flex; align-items: center;">
-            <rux-icon icon="apps" aria-controls="popup-menu-1"></rux-icon>
-            <span style="margin-left: 10px;">Click icon to see pop up</span>
-        </div>
-        <rux-pop-up-menu id="popup-menu-1" .open="${args.open}">
-            <rux-menu-item>Item 1</rux-menu-item>
-            <rux-menu-item-divider></rux-menu-item-divider>
-            <rux-menu-item value="2" style="max-width: 344px"
-                >Item 2 with an exceedingly long title that overruns the
-                width</rux-menu-item
-            >
-            <rux-menu-item disabled>Item 3 is disabled</rux-menu-item>
-            <rux-menu-item value="Item 4"
-                >Item 4 has a string value</rux-menu-item
-            >
-            <rux-menu-item href="https://www.astrouxds.com"
-                >Item 5 is an anchor</rux-menu-item
-            >
-        </rux-pop-up-menu>
+<div style="display: flex; align-items: center;">
+    <rux-icon icon="apps" aria-controls="popup-menu-1"></rux-icon>
+    <span style="margin-left: 10px;">Click icon to see pop up</span>
+</div>
+<rux-pop-up-menu id="popup-menu-1" ?open="${args.open}">
+    <rux-menu-item>Item 1</rux-menu-item>
+    <rux-menu-item-divider></rux-menu-item-divider>
+    <rux-menu-item value="2" style="max-width: 344px"
+        >Item 2</rux-menu-item
+    >
+    <rux-menu-item disabled>Item 3 is disabled</rux-menu-item>
+    <rux-menu-item value="Item 4"
+        >Item 4 has a string value</rux-menu-item
+    >
+    <rux-menu-item href="https://www.astrouxds.com"
+        >Item 5 is an anchor/action item...</rux-menu-item
```

Comment:
Changed this to an ellipsis action item, as well as removed the long overflow: ellipsis item 2 further up. The css for pop up remains unchanged, this was just for our SB example